package.json

```diff
@@ -349,7 +349,7 @@
 		"ts-loader": "^8.0.11",
 		"ts-morph": "^21.0.1",
 		"ts-node": "^10.9.1",
-		"typescript": "^5.2.2",
+		"typescript": "^5.3.3",
```

Comment:
i ran into a really weird bug with typescript where it would yell at me saying the output schema of `GET /api/pubs` was incompatible for some reason. Never happened in the editor, only when running `tsc`. Could not figure out what caused it, seemed like a false alarm, upgrading solved this.